src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -4871,18 +4870,15 @@ public static void main(String arg[]) throws Throwable {
 		fout.delete();
 		final Bucket inputBucket = new FileBucket(fin, true, false, false, false);
 		final Bucket outputBucket = new FileBucket(fout, false, true, false, false);
-		InputStream inputStream = null;
-		OutputStream outputStream = null;
-		try {
-			inputStream = inputBucket.getInputStream();
-			outputStream = outputBucket.getOutputStream();
+		try (
+			InputStream inputStream = inputBucket.getInputStream();
+			OutputStream outputStream = outputBucket.getOutputStream()
+		) {
```

Comment:
nice!

Reply:
There was already #830 (and more broadly #954 #955 and so on). I think it would be nice if we can get these into next as well.

Reply:
#830 needs a review and #955 needs changes from review, but #954 is ready to be merged — will merge it.